### ISSUE TITLE
Known phenomena?

### ISSUE THREAD

**Issue by michito744** [lightvector/KataGo#715]
The following game record is the actual game of the "Two-Headed Dragon".
[KataGo_sample_20221217_0001.txt](https://github.com/lightvector/KataGo/files/10246951/KataGo_sample_20221217_0001.txt)

White is sure to live because wins against the ko of A.
![2022-12-17](https://user-images.githubusercontent.com/53891655/208131097-c5736c75-1290-41ef-94e6-b4b9d5e54fd4.png)

In order to know what KataGo does not understand, I let the game proceed under the assumption that White moved to a. The result was unbelievable: KataGo could not recognize that the white stone in the center was dead until it was down to two liberties.
It did not understand life and death at all.

**Comment by lightvector** [comment]
Yes, this is known. All known AlphaZero-trained resnets have issues evaluating two-headed dragons, similar to ladders, it appears to be something that AlphaZero bots consistently fail to learn without intervention. I'm currently already experimenting with special training to see if it can be improved.

**Comment by lightvector** [comment]
By the way, thanks for the game record! I can also use it as a training sample too for such positions! :)

**Comment by michito744** [comment]
@lightvector Thanks for the reply.

I understand the above.
I see that KataGo does not understand the meta-structure of "the opponent's living stones can exist in an area that is completely enclosed by myself - by doing so, own moving prohibition point is taken away".

By the way, it would be great if you could also check the seki collapse in #705. 
That example also misinterprets life and death until the number of  liberties is decisively reduced.
